src/pages/Bridge/index.tsx

```diff
@@ -0,0 +1,297 @@
+import React, { useCallback, useEffect, useState } from 'react'
```

Comment:
please fix the ui
![image](https://user-images.githubusercontent.com/109076629/211133869-9bdc85b3-33b4-47fc-a085-fb6aedd636be.png)

Reply:
please check the spelling in general, it's `wallet`, not `wellet`